bot/exts/conversions/convert.py

```diff
@@ -33,7 +34,7 @@ async def convert(
         except commands.BadArgument as e:
             try:  # It might be a .SVG
                 raw_bytes = (await download_bytes(image_url)).getvalue()
-                image = rasterize_svg(raw_bytes)
+                image = await in_executor(rasterize_svg,raw_bytes)
             except BadArgument:
                 raise e  # Raise the original error
 
```

Comment:
On the line below (can't highlight since it isn't changed), it runs `image_to_file`. Look at the source for that and see if you think it needs to be run in the executor.

Reply:
Will add